### ISSUE TITLE
markdownify | strip_html - archive single

### ISSUE THREAD

**Issue by jaybe-jekyll** [mmistakes/minimal-mistakes#1460]
https://github.com/mmistakes/minimal-mistakes/blob/a8d7bf2237870fb894cd3f45968bde6a3d60969b/_includes/archive-single.html#L36

Is strip_html upon the excerpt for ensuring sanitization?

Asking because I was using markdown within excerpt with anticipation of the markdown rendered within the home layout/ archive-single list of posts:

<img width="365" alt="screen shot 2018-01-09 at 11 53 53 am" src="https://user-images.githubusercontent.com/5395940/34735099-ccd6c552-f533-11e7-84e1-3658c7e57e36.png">

The content/excerpt markdown renders:

<img width="425" alt="screen shot 2018-01-09 at 11 54 38 am" src="https://user-images.githubusercontent.com/5395940/34735123-e87e7692-f533-11e7-9594-5d785baf2bd0.png">

Presumed I would get rendered excerpt. Wanting to learn more/how/why. Thanks!

**Comment by mmistakes** [comment]
Yes that is correct. There are some edge-cases that can break the layout and/or look bad if you're not vigilant with your body content. So I choose the option of least resistance and normalize excerpts into plain-text enclosed in a single `<p></p>`.

For example on the test site I just removed the `strip_html` filter and went with something like this: `{{ post.excerpt | markdownify | remove: "<p>" | remove: "</p>" | truncate: 160 }}`

And for most of the posts it looks good, then you hit an auto-excerpt generated by Jekyll that is truncated, and it's stripping off the closing `</code>` tag... which screws up the font for everything after.

![image](https://user-images.githubusercontent.com/1376749/34737358-eb66aa86-f543-11e7-920b-a2d6902c5a49.png)

If you create custom `excerpt`s for all your posts you'll be safe. But with the theme I needed to build it in a way that works with a variety of edge-case content.
